packages/docs/package.json

```diff
@@ -7,7 +7,7 @@
   "style": "dist/docs.css",
   "unpkg": "dist/docs.bundle.js",
   "dependencies": {
-    "@blueprintjs/core": "^1.25.1",
+    "@blueprintjs/core": "^1.27.0",
```

Comment:
revert

Reply:
What's the rule of thumb with package versions in `site-landing`, `site-docs`, and `docs`? (For both dependencies and the package itself.)